<launch>
    <node name="image_cropper" pkg="ur5lego" type="image_cropper.py"/>
    <!--node pkg="rqt_image_view" type="rqt_image_view" name="image_viewer" output="screen">
        <remap from="image" to="/image_cropped"/>
    </node-->
</launch>